### PR TITLE
feat: Added support for django 32

### PR DESCRIPTION
**Pull request by Jawayria**
JIRA: https://openedx.atlassian.net/browse/BOM-2890

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         os: [ubuntu-20.04]
         python-version: ['3.8']
-        toxenv: ["django22", "django30", "quality", "docs", "pii_check"]
+        toxenv: [quality, docs, pii_check, django22, django30, django31, django32]
```
Comment (mraarif) [inline .github/workflows/ci.yml:19]: shouldn't we stop testing older Django versions now? if the tests are passing on all versions we can add tests only for 3.2

Comment (Jawayria) [reply .github/workflows/ci.yml:19]: Yes you're right

File: setup.py [hunk 2]
```diff
@@ -87,6 +87,9 @@ def is_requirement(line):
         "Development Status :: 3 - Alpha",
         "Framework :: Django",
         "Framework :: Django :: 2.2",
+        "Framework :: Django :: 3.0",
```
Comment (mraarif) [inline setup.py:90]: shouldn't this only have 3.2?

File: CHANGELOG.rst [hunk 3]
```diff
@@ -38,6 +38,10 @@ Unreleased
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 * fix analytics key in template
 
+[0.12.0] - 2021-11-2
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Added support for django 3.1 and 3.2
```
Comment (mraarif) [inline CHANGELOG.rst:43]: and this should say dropped all other versions except 3.2

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -2,10 +2,10 @@ name: Python CI
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
```
Comment (UsamaSadiq) [inline .github/workflows/ci.yml:5]: Should use same notation (`[]` or `-`) in both options (i.e. `push` and `pull_request`).

**Review by UsamaSadiq — APPROVED** [review]

**Review by mraarif — APPROVED** [review]

**Comment by edx-deployment** [comment]
A PR has been created in edx-internal to release this new version. Please review it here: https://github.com/edx/edx-internal/pull/5751

**Pull request merged**